usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -136,6 +141,7 @@
         "swap_schema": None,
         "partition_column": "published_fabs_id",
         "partition_column_type": "numeric",
+        "is_partition_column_unique": True,
```

Comment:
There isn't actually a unique constraint on the partition columns for `transaction_fabs` and `transaction_fpds` (`published_fabs_id` and `detached_award_procurement_id` respectively). Looking in the code to see if there is another way we can guarantee these are unique.

Reply:
Good point. Should we really be using those columns then? Why not just use the guaranteed unique (and indexed) `transaction_id` col for each one?

Reply:
Looks like those fields are PKs/unique on `source_assistance_transaction` and `source_procurement_transaction`, which should be 1 to 1 with fabs/fpds tables, so we're good here.

Reply:
I think you're right that `transaction_id` could be a better column for partitioning. Want to include that change? We can do a test run in staging over the weekend.